Gemfile

```diff
@@ -11,6 +11,7 @@ gemspec
 gem "hocon", '>= 1.2.5'
 gem "json-schema", '>= 2.8.0'
 gem "puma", '>= 3.12.0'
+gem "puppet", git: 'https://github.com/donoghuc/puppet.git', branch: 'GH-1288'
```

Comment:
Do we know when this will be in a release?

Reply:
Next platform release is November 19

Reply:
Did we decide on pulling from master, or waiting for release?